### DISCUSSION TITLE
Question about the "ComputeSurfaceTensionKKS"

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#18432] [Q&A General] [answered]
Dear MOOSE group,

Based on the  Laplace equation, the surface tension is 2γ/R. γ is the surface energy. 1/R is the curvature. In the "ComputeSurfaceTensionKKS" example, the calculated extra_stress in the interface is 2γ/R or γ? 

![image](https://user-images.githubusercontent.com/60105926/126906812-9af89f69-be65-4c00-abe5-22fbbfbedd5a.png)

https://mooseframework.inl.gov/source/materials/ComputeSurfaceTensionKKS.html

Thanks,

Peng

**Comment by dschwen** [comment, marked as answer]
@laagesen

**Reply by laagesen** [reply]
This form will give surface tension 2*gamma/R.

**Reply by wenpeng231024** [reply]
Got it! Thanks.

**Reply by wenpeng231024** [reply]
Can this equation reflect the curvature of the gas bubble? Because when I use this equation to calculate the surface tension for different size bubbles. I found no difference in surface tension for the different radius of the gas bubbles. I use (extra_stress_00+extra_stress_11)/2 to calculate the surface tension in 2D.

Thanks,

Peng

**Reply by dschwen** [reply]
You will find a pressure dependence on the radius.
